<launch>
  <node pkg="ptp_arm_action" type="ptp_arm_action_server.py" args="left" output="screen" name="ptp_action_left">
  	    <!--<param name="translation_tolerance" value=".015"/>-->
        <param name="rotation_tolerance" value="5"/>
        <!--<param name="timeout" value="30"/>-->
  </node>

  <node pkg="ptp_arm_action" type="ptp_arm_action_server.py" args="right" output="screen" name="ptp_action_right">
  	    <!--<param name="translation_tolerance" value=".015"/>-->
        <param name="rotation_tolerance" value="5"/>
        <!--<param name="timeout" value="30"/>-->
  </node>

</launch>
